h/static/scripts/directive/test/group-list-test.js

```diff
@@ -122,20 +102,9 @@ describe('groupList', function () {
     var shareLinks = element.find('.share-link-container');
     assert.equal(shareLinks.length, 1);
 
-    var linkField = element.find('.share-link-field');
-    assert.equal(linkField.length, 1);
-    assert.equal(linkField[0].value, GROUP_LINK);
-  });
-
-  it('should toggle share link on click', function () {
-    var element = createGroupList();
-    var toggleLink = element.find('.share-link-toggle');
-    var expander = element.find('.share-link-expander');
-    assert.ok(isElementHidden(expander));
-    toggleLink.click();
-    assert.ok(!isElementHidden(expander));
-    toggleLink.click();
-    assert.ok(isElementHidden(expander));
+    var link = element.find('.share-link');
+    assert.equal(link.length, 1);
+    assert.equal(link[0], GROUP_LINK);
```

Comment:
The diff here is confusing. This pr deletes some no longer relevant tests, does not add any test code.

The neat trick here is used a few times throughout this test file